modules/ROOT/pages/getting-started-proxy.adoc

```diff
@@ -1,89 +1,96 @@
-= Quick start: Manage an API and set up a proxy for it
+= Getting Started with Managing an API 
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this walkthrough, we are showing how to create an API that can be managed through API Manager and set up a proxy for it that
-runs in CloudHub.
+You can use API Manager to apply policies and to add contracts, alerts, and SLA tiers to your API instance. Additionally, you can configure a proxy for your API that runs in CloudHub. Although you can also deploy your API proxy using Mule runtime engine (Mule) and Hybrid, deploying your API proxy directly to CloudHub is the fastest and easiest.
 
-== Creating an API asset in Exchange
+You can manage your API using the following steps:
 
-In order to have an API that can be managed through API Manager, the first step consists in creating an API asset in Anypoint Exchange.
+. <<create_an_api, Create an API asset in Exchange.>>
+. <<import-api-to-API-Manager, Import your API to API Manager.>>
+. <<configure_and_deploy_to_cloudhub, Configure an API proxy and deploy it to CloudHub.>>
 
-. Go to Anypoint Exchange https://anypoint.mulesoft.com/exchange
-. Click in `New Asset` button
+[[create_an_api]]
+== Create an API Asset in Exchange
+
+Before you can manage an API in API Manager, you must first create it in Anypoint Exchange:
```

Comment:
API Instance in API Manager, API Asset in Exchange.

Reply:
Done! Thanks!